__init__.py

```
@@ -473,56 +470,9 @@ def show_time(self, display_time: str):
-                                             x=xoffset, refresh=False)
-                if c == ":":
-                    xoffset += 2  # colon is 1 pixels + a space
-                else:
-                    xoffset += 4  # digits are 3 pixels + a space
-
-        self.enclosure.mouth_display(img_code="CIAAAA", x=29,
-                                     refresh=False)
-        time.sleep(5)
-        self.enclosure.mouth_reset()
-        self.enclosure.activate_mouth_events()
+        LOG.debug(f"sending time to mk1 {text}")
```

Ruff

Undefined name `text`

(F821)